Gopkg.lock

```diff
@@ -316,6 +316,12 @@
   packages = ["."]
   revision = "ed27b6fd65218132ee50cd95f38474a3d8a2cd12"
 
+[[projects]]
+  name = "github.com/status-im/go-web3js"
```

Comment:
Can you add a short description of this dependency right into this file?
Maybe with even a link to README (looks like overkill, but it makes it more convenient when one needs to regenerate and update dependency).